proposed/2021/NuGet.orgTFMs.md

```diff
@@ -0,0 +1,96 @@
+- GitHub Issue (https://github.com/NuGet/NuGetGallery/issues/8387)
+- GitHub PR ()
+
+## Summary
+
+When a developer opens the package details page or tab on NuGet.org or NuGet rooling, they will see what frameworks the package can support. 
+
+## Motivation
+
+### Problem: What problem is this solving? 
+
+Developers cannot use the package details page to determine what frameworks a NuGet package can support. In 2021, we live in a world where there are three evolutions of .NET coexisting on NuGet; .NET Framework, .NET Core, and .NET 5.  
+
+Developers should be able to determine whether a package works for their project without having to first download it or rely on the author’s documentation. 
+
+### Why: How do we know this is a real problem and worth solving? 
+
+Finding a package that is compatible with your project is a challenge for the average developer in the .NET ecosystem. Over 21% of developers fail to install a package today. One of the top NuGet errors issued in telemetry is [NU1202](https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu1202), which shows that we allow developers to install packages that are not compatible with their project or solution’s target framework. 
+
+### Success: How do we know if we’ve solved this problem? 
+
+- By measuring the change in the percentage of successful installs and lowering the amount of [NU1202](https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu1202) errors in telemetry. 
+- By measuring the usage of target framework & supported platform filters on NuGet.org & Visual Studio. 
+- By gauging excitement/disappointment after blogging about this feature & analyzing sentiment on Twitter/GitHub/DevCom/etc. 
+
+### Audience: Who are we building for? 
+
+We are building this for the .NET developer who is browsing & making decisions about including a NuGet package into their project based on target framework & supported platform criteria. 
+
+## Explanation
+
+### Functional Explanation
+
+#### Context and Scope 
+When a developer accesses the details page for a NuGet package, they will see information on target frameworks in a dropdown menu. The framework information will be surfaced from the package. In cases where versions are not specified (such as .net) or other unexpected scenarios, ‘Version Not Specified’ or similar messages will be surfaced. 
+
+##### Minimal Requirements 
+- List of Frameworks on NuGet.org
+
+##### Non-Requirements 
+- List of Platforms on NuGet.org
+  - Users interviewed did not express need for supported platform information 
+  - Cannot be surfaced directly from package data and would require additional layer of soft calculation from engineering (no clear mapping scheme currently exists between target frameworks and supported platforms) 
+  - Can be revisited in future 
+- Filters for Frameworks on NuGet.org
+    - Users interviewed were ambivalent on filters, requires further investigation. Can be revisited in future 
```

Comment:
Does this mean users were ambivalent on searching by a specific target framework?

Reply:
Frameworks are really platforms pre .NET 5. .NET 5+ we can leverage platforms more. 

I think there's a hidden requirement here in which the "platform is implied by the TFM". At least from many people I've talked to, a TFM is synonymous with a platform today but will change in NET 6 drastically.

Reply:
In my eyes we do the following:

TFM -> Platforms -> Runtimes

Reply:
I believe the question was about the priority of filtering, and I'll call this good for this iteration. My understanding is that this will be spec'd and implemented separately from the Package Details UX.